internal/monitors/collectd/df/metadata.yaml

```diff
@@ -20,26 +25,32 @@ monitors:
     included: false
     name: df_inodes.free
     type: gauge
+    group: ReportInodes
```

Comment:
Let's make these camel case so `reportInodes` just for consistency with some other configs I had done.